README.rst

```diff
@@ -24,187 +31,207 @@ check out `tequila on Github <https://github.com/caktus/tequila>`_.
+Ansible supports setting variables in many places.  Here are some
+recommended conventions:
 
-    tequila_dir: Directory where tequila was installed.  The ``roles`` subdirectory
-    of this is added to the roles path.
+* Variables telling Ansible how to connect to a particular host go into
+  the inventory file, on the same line as the first mention of that
+  host.
 
-    env_name: Name of the environment being deployed.
+* Each environment directory and the playbooks directory may have
+  either a ``group_vars/`` subdirectory or a ``host_vars/``
+  subdirectory, or both.  These directories can then contain files or
+  directories named for the group (in ``group_vars/``) or the host (in
+  ``host_vars/``), which will then have their contents automatically
+  included when that environment is deployed or playbook is executed.
 
-    local_project_dir: The current directory when deploy was invoked.
+  The special group name ``all`` will match all groups.
 
-* Variables that are global to the project go in ``inventory/group_vars/all/vars.yml``
-  and ``inventory/group_vars/all/secrets.yml``::
+* Variables that are global to the project go in
+  ``deployment/playbooks/group_vars/all.yml``, or if more modularity
+  is needed, ``deployment/playbooks/group_vars/all/<filename>.yml``::
 
     ---
-    # file: inventory/group_vars/all/vars.yml
+    # file: deployment/playbooks/group_vars/all.yml
     project_name: our_project
+    python_version: 3.4
+    less_version: 2.1.0
+    postgres_version: 9.3
 
-* Variables that apply to all servers in an environment go in
-  ``inventory/group_vars/<envname>/vars.yml`` and
-  ``inventory/group_vars/<envname>/secrets.yml``::
+* Variables that apply to all servers in a particular environment go
+  in ``deployment/environments/<envname>/group_vars/all.yml`` or
+  ``deployment/environments/<envname>/group_vars/all/<filename>.yml``::
 
     ---
-    # file: inventory/group_vars/staging/vars.yml
+    # file: deployment/environments/staging/group_vars/all/vars.yml
     domain: project-staging.example.com
+    repo:
+      url: git@github.com:caktus/caktus-website.git
+      branch: develop
+
+* Variables whose values should be secret (such as passwords and API
+  keys) should be kept in separate variable files, named by convention
+  ``secrets.yml``, that get encrypted using ``ansible-vault``.  Only
+  secret values should be kept in encrypted files.  Since such secrets
+  will probably be specific to the environment, we recommend that they
+  be placed in
+  ``$(PWD)/deployment/environments/<envname>/all/secrets.yml``.  All
+  variables defined in such encrypted files should then be exposed in
+  a corresponding non-encrypted file ::
 
-* Variables whose values should be secret should be declared in the same
-  files as other variables, depending on their scope, but their value
-  should be set to ``{{ secret_<varname> }}``::
+    ---
+    # file: deployment/group_vars/staging/secrets.yml
```

Comment:
this path doesn't match the one in the paragraph above (`deployment/environments/<envname>/all/secrets.yml`)